docs/usage.md

```diff
@@ -62,7 +62,7 @@ An [example samplesheet](../assets/samplesheet.tsv) has been provided with the p
 The typical command for running the pipeline is as follows:
 
 ```console
-nextflow run nf-core/mhcquant --input 'samples.tsv' --outdir <OUTDIR> --fasta 'SWISSPROT_2020.fasta' --allele_sheet 'alleles.tsv' -profile docker
```

Comment:
Are we slowly removing this feature?

Reply:
That is the idea. At least let us not mention it in usage as typical run